test/test_package.py

```diff
@@ -558,6 +558,21 @@ def test_pickle_mocked(self):
         with self.assertRaises(NotImplementedError):
             hi.load_pickle('obj', 'obj.pkl')
 
+    @skipIf(version_info < (3, 7), 'mock uses __getattr__ a 3.7 feature')
```

Comment:
I think we need more exhaustive testing for the pickle analyzer, since it should run on every export and raise errors that could block users.

At the least we should ensure we have coverage of the pickle bytecodes we know about